index.html

```diff
@@ -5104,7 +5104,7 @@ <h4 id=el-li>`li`</h4>
     <tr>
       <th>Comments</th>
       <td>
-        If `li` element is not a child of <a data-cite="html">`ol`</a> , <a data-cite="html">`menu`</a> or <a data-cite="html">`ul`</a>, or if the containing
+        If `li` element is not an accessibility child of an <a data-cite="html">`ol`</a> , <a data-cite="html">`menu`</a> or <a data-cite="html">`ul`</a>, or if the containing
```

Comment:
only comment -- should this link to the ARIA spec def of accessibility child? (Guessing it should link to 7.3)

Reply:
yes.  tbh i couldn't remember when i was writing this if that def existed yet

Reply:
I think you can link with `[=accessibility child=]` per: https://respec.org/xref/?term=accessibility+child

```suggestion
        If `li` element is not an [=accessibility child=] of an <a data-cite="html">`ol`</a> , <a data-cite="html">`menu`</a> or <a data-cite="html">`ul`</a>, or if the containing
```